.github/workflows/validate-pipelines.yml

```diff
@@ -0,0 +1,62 @@
+name: Validate Deployment Pipelines
```

Comment:
nit: this file uses 4 spaces rather than 2, can we move it to 2 for consistency with other workflows in the repo?